src/mint_json.cr

```diff
@@ -203,11 +206,28 @@ module Mint
       }
     end
 
-    # Parsing external javascripts
+    # Parsing external assets (JavaScripts, CSS)
     # --------------------------------------------------------------------------
-    json_error MintJsonExternalJavascriptNotExists
+
+    json_error MintJsonExternalInvalid
+
+    def parse_external_assets
+      @parser.read_object do |key|
+        case key
+        when "javascripts"
```

Comment:
I'd consistently use either form for both entries:
- `javascripts` + `stylesheets`
- `js` + `css`

Reply:
@gdotdesign Should we stick to `stylesheets` instead of `css`?

Reply:
`stylesheets` is nice, maybe in the future we could preprocess `sass` if required, the name is not limited to `css`.

`js` is not descriptive.

Reply:
yeah we should use `stylesheets` don't know why I didn't think of that :man_facepalming:

Reply:
More coffee is always the answer :smile: